roles/zitadel/templates/values.yaml.j2

```diff
@@ -0,0 +1,61 @@
+---
+zitadel:
+  masterkey: {{ zitadel_master_key}}
+  secretConfig:
+    Database:
+      Postgres:
+        User:
+          Password: {{ zitadel_db_password }}
+        Admin:
+          Password: {{ zitadel_db_password }}
+  configmapConfig:
+    FirstInstance:
+      InstanceName: MetalStack
```

Comment:
Would it be possible to use `metal-stack` or is this casing a typical convention?